nanoFramework.System.Net.Http/Http/Headers/KnownHeaders.cs

```diff
@@ -0,0 +1,14 @@
+﻿//
+// Copyright (c) .NET Foundation and Contributors
+// Portions Copyright (c) Microsoft Corporation.  All rights reserved.
+// See LICENSE file in the project root for full license information.
+//
+
+namespace System.Net.Http.Headers
+{
+    internal static class KnownHeaders
+    {
+        //public static readonly KnownHeader PseudoStatus = new(":status", HttpHeaderType.Response);
```

Comment:
either remove fully either add few of the known header here.
Thinking about performance of creating all those known headers all the time, does it makes sense to have this as a static somewhere? Maybe even lazy loaded static class (can be internal and exposed with similar API as full framework)

Reply:
Removed. We don't need it actually.
Known headers it's already a string list. We are creating the ones needed when creating on HttpClient